src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,8 @@
 
 public class Application {
```

Comment:
Don't forget to add an access modifier to your class. The default access modifier may not be suitable for your needs and it's considered good practice to explicitly define the access level.